lib/src/data/datasource/remote/movie_list_api_service.dart

```diff
@@ -0,0 +1,37 @@
+import 'dart:convert';
+import 'package:http/http.dart';
+
+import '../../../core/util/constants.dart';
+import '../../model/movie_page_model.dart';
+import 'i_api_service.dart';
+
+class MovieListApiService implements IApiService {
+  final String endpoint;
+  final String exceptionMessage = 'Failed to load the movies';
+  final int correctStatusCode = 200;
```

Comment:
Instead to add a const you can use HttpStatus.OK

Reply:
solved